mods/ts/sequences/misc.yaml

```diff
@@ -241,12 +260,14 @@ moveflsh:
 	idle: ring
 		Length: *
 		Tick: 30
+		ZOffset: 2047
 
 wake:
 	idle: wake2
 		Length: *
 		Tick: 180
 
+		ZOffset: 2047
```

Comment:
The new line above this doesn't look correct. We probably want a new line below instead.